src/main/java/org/takes/facets/auth/PsBasic.java

```diff
@@ -284,6 +286,9 @@ public Default(final String... users) {
          * @param pwd Password.
          * @return Opt with URN or empty if there is no such login-password
          *  pair.
+         * @todo #744:30min `null` constant usage in the function
```

Comment:
@Iprogrammerr We do not need this puzzle here, it is being overlapped by the puzzle defined above, please remove it